files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
Since the github repository is only supposed to provide source files (not production) the only way to handle this is having a release script to publish  the dist files for each tag. Unfortunately we do not have a release script yet.

I thought jsDelivr could handle unminified files, because cdnjs [also](https://github.com/cdnjs/cdnjs/pull/4713) [does it](https://github.com/cdnjs/cdnjs/commit/4d175c615fa6f123115b15118187c66e21ee39a3), but if there's no solution I think we may pass jsDelivr for now until we have a proper release script or when jsDelivr support file minification from source.